ide/editor.completion/src/org/netbeans/modules/editor/completion/CompletionImpl.java

```diff
@@ -1414,16 +1414,18 @@ boolean hideDocumentationPane(boolean documentationOnly) {
      * May be called from any thread but it will be rescheduled into AWT.
      */
     public void showToolTip() {
-        if (!SwingUtilities.isEventDispatchThread()) {
-            // Re-call this method in AWT if necessary
-            SwingUtilities.invokeLater(new ParamRunnable(ParamRunnable.SHOW_TOOL_TIP));
-            return;
-        }
-
-        if (ensureActiveProviders()) {
-            toolTipCancel();
-            toolTipQuery();
-        }
+		if (CompletionSettings.getInstance(getActiveComponent()).completionDisplayTooltip()) {
```

Comment:
nitpick: you are using tabs here, please use spaces if possible (but read other message first).

Reply:
yeah that's just what netbeans did, i couldn't find a way to change the settings.

Reply:
under editor->formatting. Expand tabs with spaces is enabled by default.